shared/js/initApp.js

```diff
@@ -172,21 +172,64 @@ dashboard.saveProject = function(callback) {
       location.hash = dashboard.currentApp.id = data.id;
       callbackSafe(callback, data);
     });
-    dashboard.showProjectHeader();
+    if (!dashboard.currentApp.hidden) {
+      dashboard.showProjectHeader();
+    }
   }
 };
 
+dashboard.loadEmbeddedProject = function(projectTemplateLevelName) {
+  var deferred = new $.Deferred();
+  // get all projects (TODO: filter on server side?)
+  storageApps().all(function(data) {
+    if (data) {
+      // find the one that matches this level
+      var projects = $.grep(data, function(app) {
```

Comment:
Again, we're doing this client-side vs. having a where-like clause in the query.  Fine for now, but we should fix this once the API supports filtering.